src/http-auth-interceptor.js

```diff
@@ -5,6 +5,12 @@
  * (c) 2012 Witold Szczerba
  * License: MIT
  */
+
+/* commonjs package manager support (eg componentjs) */
+if (typeof module !== "undefined" && typeof exports !== "undefined" && module.exports === exports){
+  module.exports = 'http-auth-interceptor';
```

Comment:
Excuse my lack of knowledge here, but what's the point of exporting just the name of the module, as a string?